### ISSUE TITLE
Python crashes on empty charts

### ISSUE THREAD

**Issue by ubershmekel** [kroitor/asciichart#32]
Trying to chart with an empty list shouldn't crash.

```
import asciichartpy

asciichartpy.plot([])
```

Causes this:

```
Traceback (most recent call last):
  File "terminal_chart.py", line 18, in <module>
    print(asciichartpy.plot(losses, chart_config))
  File "/Users/lib/python3.7/site-packages/asciichartpy/__init__.py", line 25, in plot
    ratio = height / interval
ZeroDivisionError: float division by zero
```

**Comment by pkazmier** [comment]
#38 fixes this btw.

**Comment by kroitor** [comment]
Thank you so much! Merged and fixed in `asciichartpy` 1.5.10!

https://pypi.org/project/asciichartpy/
